pygfx/renderers/wgpu/shaders/lineshader.py

```diff
@@ -125,13 +125,31 @@ def bake_function(self, wobject, camera, logical_size):
                 return
             self._positions_hash = positions_hash
             vertex_array = positions_array
-        elif wobject.material.thickness_space == "world":
-            vertex_array = la.vec_transform(positions_array, wobject.world.matrix)
-        else:  # wobject.material.thickness_space == "screen":
-            xyz = la.vec_transform(
-                positions_array, camera.camera_matrix @ wobject.world.matrix
-            )
-            vertex_array = xyz[:, :2] * (0.5 * np.array(logical_size))
+        else:
+            # Prep
+            finites = np.isfinite(positions_array).all(axis=1)
+            has_non_finites = finites.sum() != finites.size
```

Comment:
I think `finites.any()` is sufficient and much faster since it can exit early

Reply:
Good one.

Reply:
Oh, actually, it needs to know if there are any non-finites. So it should be `not finites.all()` 😄

Reply:
lol, apparently was looking at the comment before it was edited.